src/main/java/edu/kis/powp/command/CommandFactory.java

```diff
@@ -0,0 +1,38 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class CommandFactory {
+    private DriverManager driverManager;
+    private int x;
+    private int y;
+
+    public CommandFactory(DriverManager driverManager, int x, int y) {
+        this.driverManager = driverManager;
+        this.x = x;
+        this.y = y;
+    }
```

Comment:
Creating a factory instance fixed on specific starting coordinates is not a very convincing idea.